tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "incremental": true,
+    "target": "ESNext",
```

Comment:
not sure if targeting `ESNext` is a good idea: https://compat-table.github.io/compat-table/esnext/